frontend/src/scenes/data-management/events/DefinitionHeader.tsx

```diff
@@ -113,15 +124,15 @@ function RawDefinitionHeader({
                 value={definition.name ?? ''}
```

Comment:
I think it'd be good to shrink the hover-able area here to just the text (or maybe even an info bubble or an explicit click). I found myself accidentally triggering the popup pretty frequently, and not being able to get to the one I wanted.

https://user-images.githubusercontent.com/4813045/157979999-074d1547-5ac6-41bc-9777-047df2638403.mov

Reply:
good catch, didn't realize that the hover area took up the full width of the container